src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,99 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl registrationService;
+    private static User testedUser;
+
+    @BeforeEach
+    void setUp() {
+        testedUser = new User();
+        Storage.people.clear();
+        testedUser.setAge(27);
+        testedUser.setLogin("thisIsUserLogin");
+        testedUser.setPassword("password");
+    }
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    public void  register_user_Ok() {
+        assertNotNull(registrationService.register(testedUser));
+    }
+
+    @Test
+    void register_userExist_notOk() {
+        registrationService.register(testedUser);
+        assertThrows(RuntimeException.class, () -> registrationService.register(testedUser));
+    }
+
+    @Test
+    void register_returnNull_notOk(){
+        assertNotEquals(registrationService.register(testedUser), null);
+    }
+
+    @Test
+    public void register_ageLessThan18_notOk() {
+        testedUser.setAge(16);
```

Comment:
```suggestion
        testedUser.setAge(17);
```
краще перевіряти крайні випадки